### PR TITLE
Pure sentinel

### PR DESCRIPTION
**Pull request by benjaminbollen**
split off statistics to separate repository flow
split sentinel into clean sentinel: only merge identical messages
   and account_sentinel
clean out implementation for pure sentinel

[Review on Reviewable](https://reviewable.io/reviews/maidsafe/sentinel/7)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,6 +14,13 @@ homepage = "http://maidsafe.net"
 [dependencies]
 rustc-serialize = "*"
 cbor = "*"
-sodiumoxide = "0.0.4"
+# sodiumoxide = "0.0.4"
```
Comment (benjaminbollen) [inline Cargo.toml:17]: I will test now; whether we can switch to sodiumoxide v0.0.5

File: Cargo.toml [hunk 2]
```diff
@@ -14,6 +14,13 @@ homepage = "http://maidsafe.net"
...
 rand = "*"
 accumulator = "0.0.1"
+
+[dependencies.flow]
+git = "https://github.com/benjaminbollen/flow.git"
```
Comment (benjaminbollen) [inline Cargo.toml:22]: this repository flow should move (sooner or later) to maidsafe

**Comment by dirvine** [comment]
Reviewed files:
- Cargo.toml @ r1
- src/account_sentinel.rs @ r1
- src/frequency.rs @ r1
- src/lib.rs @ r1
- src/sendable.rs @ r1
- src/sentinel.rs @ r1

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/sentinel/7)

File: Cargo.toml (hunk 1 above)
Comment (benjaminbollen) [inline Cargo.toml:17]: sodiumoxide v0.0.5 still has a build error, so unstable.

**Pull request merged**